Readme.md

```diff
@@ -78,14 +78,25 @@ irb(main):001:0>
 
 ## Custom Rails prompt
 
-If you want to include the current Rails environment and project name in the pry prompt, put the following lines in your project's `.pryrc`:
+If you want to permanently include the current Rails environment and project name
+in the Pry prompt, put the following lines in your project's `.pryrc`:
 
 ```ruby
-if defined?(PryRails::RAILS_PROMPT)
-  Pry.config.prompt = PryRails::RAILS_PROMPT
+Pry.config.prompt = Pry::Prompt[:rails]
+```
+
+If `.pryrc` could be loaded without pry-rails being available or installed,
+guard against setting `Pry.config.prompt` to `nil`:
+
+```ruby
+if Pry::Prompt[:rails]
+  Pry.config.prompt = Pry::Prompt[:rails]
```

Comment:
Just in case this got missed in force push:

I like this but would recommend providing instructions for pre Pry v0.12.0 as well:

```ruby
# For Pry v0.12.0+
if Pry::Prompt[:rails]
  Pry.prompt = Pry::Prompt["rails"][:value]
end

# For Pry < v0.12.0
if defined?(PryRails::Prompt)
  Pry.config.prompt = Pry::Prompt::MAP["rails"][:value]
end
```